README.md

```diff
@@ -79,6 +79,13 @@ By default, only the first failing test log is printed to the console
 (and all test logs are captured on disk). Pass `--show-errors` to
 print all error messages.
 
+### Using Alcotest with opam and Dune
+
+Add `(alcotest :with-test)` to the `depends` stanza of your
+`dune-project` file, or `"alcotest" {with-test}` to your opam file.
+You can then declare your test and link with Alcotest:
+`(test (libraries alcotest …) …)`.
```

Comment:
I did not use it like this personally. I had a dune file like this:

```scheme
(executable
 (name this_package)
 (libraries alcotest))

(rule
 (alias runtest)
 (package this_package)
 (action
  (run test.exe)))
```

Reply:
I also had to run `opam install alcotest`

Reply:
The big `(rule (alias runtest) (action ...))` stanza can be shortened by the `test` stanza. I'll add a link to the Dune documentation. 
Adding alcotest to the `with-test` dependencies means that `opam install --with-test .` will install all the dependencies you need to run your tests. I'll add that too.
Thanks for the feedback!

Reply:
oh interesting, I didn't know that, thanks!